README.md

```diff
@@ -140,8 +140,13 @@ ENUMS::USER_ROLES.constants
 
 ###### Prerequisites
 
-- ruby version `2.6.2` via [rbenv](https://github.com/rbenv/rbenv)
-- `brew install graphviz`
+- ruby version `2.6.3` via [rbenv](https://github.com/rbenv/rbenv)
+- NodeJS version `<12.0.0` via [nvm](https://github.com/nvm-sh/nvm) with yarn installed globally
+- graphviz `brew install graphviz` or `sudo apt-get install graphviz`
+- PostgreSQL 11
+- Redis
+
+You must create a (superuser) role with the name of your OS user in your postgres configuration in order to run db operations (e.g. testing and development).
```

Comment:
I was able to get up and running with `brew install postgre` and then use the following without running it as a full service.

Start: `pg_ctl -D /usr/local/var/postgres start`
Stop:`pg_ctl -D /usr/local/var/postgres stop`

Reply:
(no configuration required)